CONFIGURATION.md

```diff
@@ -174,6 +174,9 @@ validate_additional_rrs:
 # The size of the payload.
 [ payload_size: <int> ]
 
+# The source IP address.
+[ source_address: <string> ]
```

Comment:
Would it be possible to support the interface name too? If there's DHCP involved you may not know the IP.

Reply:
Should be possible. Resolve the interface name to interface index first from `net.InterfaceByName()`, then set `IfIndex` in the `ipv6.ControlMessage` struct.

Reply:
@brian-brazil how about a source_iface attribute in another PR?